### DISCUSSION TITLE
articles/wisdom-of-dengxiaoping

### DISCUSSION THREAD

**Discussion by giscus** [hutusi/hutusi.github.com#161] [Comments]
# articles/wisdom-of-dengxiaoping

《邓小平时代》是去年读完的，后来又在 iBooks 上买了原版有声书听了一遍。最近因为居家二十余天，便翻了翻回顾了下。作者傅高义是位出生美国的犹太人，精通中文和日文，是最为著名的中国问题专家。《邓小平时代》英文原版名为 Deng Xiaoping and the Transformation of China, 于 2011年出版，简体中文版于2013年在大陆出版，有删节，不过作者认为保留了他想表达的核心内容。

https://hutusi.com/articles/wisdom-of-dengxiaoping

**Comment by bearfm** [comment]
妈呀！这么高深的著作，膜拜一下

**Reply by hutusi** [reply]
是本好书~

**Comment by acegao9527** [comment]
原版真的很精彩，感叹老外比我们国内的研究的全面多了，十年成一书

**Reply by hutusi** [reply]
看了一些外国人写的关于中国的书，发现都写的很好，可能真的应了那句话“旁观者清”~

**Comment by 2Broear** [comment]
等老先生对中国老百姓做的贡献太大了，有机会一定好好读一读

**Reply by hutusi** [reply]
书不错，学习伟人的智慧
